### ISSUE TITLE
d_loss_real calculated against 0.9?

### ISSUE THREAD

**Issue by BrianSantoso** [llSourcell/Generative_Adversarial_networks_LIVE#9]
In cell 3 d_loss_real is defined as 

`d_loss_real = tf.reduce_mean(tf.nn.sigmoid_cross_entropy_with_logits(logits=Dx, labels=tf.fill([batch_size, 1], 0.9)))`

Why are the labels 0.9 as opposed to 1?
